kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -36,7 +40,7 @@
       },
       channelId: {
         type: String,
-        default: '',
+        default: '7199dde695db4ee4ab392222d5af1e5c',
```

Comment:
If you merge the latest from master, I think you'll get both the session stuff and the channel switching stuff.